packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -59,67 +316,37 @@ export default util.createRule<Options, MessageIds>({
         properties: {
           default: {
             oneOf: [
-              {
-                enum: ['never'],
-              },
-              {
-                type: 'array',
-                items: {
-                  enum: allMemberTypes,
-                },
-              },
+              neverConfig,
+              allMemberTypesArrayConfig,
+              allMemberTypesObjectConfig,
             ],
           },
           classes: {
             oneOf: [
-              {
-                enum: ['never'],
-              },
-              {
-                type: 'array',
-                items: {
-                  enum: allMemberTypes,
-                },
-              },
+              neverConfig,
+              allMemberTypesArrayConfig,
+              allMemberTypesObjectConfig,
             ],
           },
           classExpressions: {
             oneOf: [
-              {
-                enum: ['never'],
-              },
-              {
-                type: 'array',
-                items: {
-                  enum: allMemberTypes,
-                },
-              },
+              neverConfig,
+              allMemberTypesArrayConfig,
+              allMemberTypesObjectConfig,
             ],
           },
           interfaces: {
             oneOf: [
-              {
-                enum: ['never'],
-              },
-              {
-                type: 'array',
-                items: {
-                  enum: ['field', 'method', 'constructor'],
-                },
-              },
+              neverConfig,
+              allMemberTypesArrayConfig,
+              allMemberTypesObjectConfig,
```

Comment:
this has loosened the interfaces config from `Array<'field', 'method', 'constructor'>` to be the same as the others, is this intentional.

Reply:
No, this is a mistake. I will change this once we've decided on the final API

Reply:
I've changed the config for `interfaces` and `typeLiterals` to only accept `'signature' | 'field' | 'method' | 'constructor'`